src/components/TutorialPage.js

```diff
@@ -40,10 +43,11 @@ const TutorialWrapper = styled.div`
   padding: 0 15px 20px;
   place-items: center;
   width: 90%;
+  height: auto;
 
   img {
     border-radius: 2px;
-    margin-top: 25px;
+    margin: 10px 10px;
```

Comment:
`margin: 10px ;`

Reply:
this works for margin-top and margin-bottom?